src/AppBar/AppBar.js

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
 import IconButton from '../IconButton';
 import NavigationMenu from '../svg-icons/navigation/menu';
-import getMuiTheme from '../styles/getMuiTheme';
 import Paper from '../Paper';
 import propTypes from '../utils/propTypes';
 import warning from 'warning';
 
-function getStyles(props, state) {
+function getStyles(context) {
```

Comment:
Personally, I think we should just maintain the method signature for consistency across the codebase `(props, context, state)`. If I was reading a render method and saw context was being passed as first argument to the method, my gut would think it's a bug.